spring-cloud-stream-binder-kinesis/src/test/java/org/springframework/cloud/stream/binder/kinesis/LocalKinesisResource.java

```diff
@@ -53,7 +49,6 @@ protected void obtainResource() throws Exception {
 		System.setProperty(SDKGlobalConfiguration.AWS_CBOR_DISABLE_SYSTEM_PROPERTY, "true");
 
 		this.resource = AmazonKinesisAsyncClientBuilder.standard()
-				.withCredentials(mock(AWSCredentialsProvider.class))
```

Comment:
I'm not sure why removing credentials doesn't work in your environment, but it seems using the `AWSStaticCredentialsProvider` also works for me - I'll add it since it seems to be a common denominator.